spec/my_enumerable_module_spec.rb

```diff
@@ -0,0 +1,43 @@
+require ('./my_enumerable_module.rb')
+
+describe Enumerable do 
```

Comment:
You should write some tests in here that *don't* use arrays. Think about the requirements for using enumerable and see if you can't implement that and then use your `my_xxx` methods.

Reply:
okay! thanks for the suggestion!